auth0_flutter/android/build.gradle

```diff
@@ -26,7 +26,9 @@ apply plugin: 'kotlin-android'
 
 android {
     compileSdkVersion 31
-
+    if (project.android.hasProperty("namespace")) {
+        namespace 'com.auth0.auth0_flutter'
```

Comment:
Since we're using the same value for `project.group`, maybe we can just reference it here?